view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -124,6 +120,15 @@ define(
                 return this;
             },
 
+            /**
+             * Get installment min amount from capability
+             *
+             * @returns {number}
+             */
+            getInstallmentMinLimit: function () {
+                return checkoutConfig.omise_installment_limit_amount.min / 100;
```

Comment:
The denominator depends on the currency. Can we do something similar to https://github.com/omise/omise-woocommerce/blob/3290c751e95a6b872c77d76cc64b5b3d26d54be2/includes/gateway/class-omise-payment-installment.php#L82

Reply:
I have added `currencyAmountFormat` and wrap to it. 
I did not use the same name as wooCommerce since there is already `omiseAmountFormat` which do the opposite of `currencyAmountFormat`.